src/main/java/io/icker/factions/ui/InfoGui.java

```diff
@@ -0,0 +1,247 @@
+                                        ),
+                                Text.literal(String.valueOf(maxPower))
+                                        .setStyle(
+                                                Style.EMPTY
+                                                        .withItalic(false)
+                                                        .withColor(Formatting.GREEN)
+                                        )
+                                        .append(Text.literal(" max faction power")
+                                                .setStyle(
+                                                        Style.EMPTY
+                                                                .withItalic(false)
+                                                                .withColor(Formatting.GRAY)
+                                                )
+                                        )
+                        )
+                )
+        );
+
+        // Allies info
+        List<Text> allies = faction.getMutualAllies().isEmpty() ?
+                List.of(Text.literal("No allies.")
+                        .setStyle(
+                                Style.EMPTY
+                                        .withItalic(false)
+                                        .withColor(Formatting.GRAY)
+                        )
+                ) :
+                faction.getMutualAllies().stream()
+                        .map(rel -> Faction.get(rel.target))
+                        .map(fac -> Text.of(fac.getColor() + fac.getName()))
+                        .toList();
+        this.setSlot(3, new GuiElementBuilder(Items.PLAYER_HEAD)
+                .setSkullOwner(Icons.GUI_CASTLE_ALLY)
+                .setName(Text.literal("Ally factions ").append(Text.literal(String.format("(%s)", faction.getMutualAllies().size())).formatted(Formatting.GREEN)))
+                .setLore(allies)
+        );
+        // Enemies info
+        List<Text> enemies = faction.getEnemiesWith().isEmpty() ?
+                List.of(Text.literal("No enemies.")
+                        .setStyle(
+                                Style.EMPTY
+                                        .withItalic(false)
+                                        .withColor(Formatting.GRAY)
+                        )
+                ) :
+                faction.getEnemiesWith().stream()
+                        .map(rel -> Faction.get(rel.target))
+                        .map(fac -> Text.of(fac.getColor() + fac.getName()))
+                        .toList();
+        this.setSlot(4, new GuiElementBuilder(Items.PLAYER_HEAD)
+                .setSkullOwner(Icons.GUI_CASTLE_ENEMY)
+                .setName(Text.literal("Enemy factions ").append(Text.literal(String.format("(%s)", faction.getEnemiesWith().size())).formatted(Formatting.GREEN)))
+                .setLore(enemies)
+        );
+
+        if (Command.Requires.isOwner().test(player.getCommandSource())) {
```

Comment:
You could also use this trick in admin instead of a seperate "factions.admin.gui" permission. I'm fine with which ever you choose.